src/main/webapp/js/ideas/ideas.controller.js

```diff
@@ -30,6 +30,16 @@
                 vm.ideas = ideas;
             });
         });
+
+        vm.details = function(idea){
+            $scope.ideaDetail = {
+                title:idea.title,
+                description:idea.description,
+                createdAt:idea.createdAt
+            };
+            detailsService.setData($scope.ideaDetail);
```

Comment:
In this case it does not make sense to use the service. Variable data is enough.